### DISCUSSION TITLE
Postgresql hook connection

### DISCUSSION THREAD

**Discussion by 708kelly** [great-expectations/great_expectations#6252] [Community Use Cases]
hi teams,
  I want to use it on airflow. Now, we use postgresql hook as connection. I had read the document and great_expectations/datasource/sqlalchemy_datasource.py. Now, it only provide use url to connect db. Is it possible to add a new connection approach or any recommend way to do it?
ref: https://stackoverflow.com/questions/61528860/sqlalchemy-engine-from-airflow-database-hook
thanks,
Kelly

**Comment by rdodev** [comment]
Hey @708kelly 👋 

I will convert this issue into a discussion and point the engineering team to it.

**Comment by joshua-stauffer** [comment]
hi @708kelly, the [GX Airflow Provider](https://github.com/great-expectations/airflow-provider-great-expectations) was recently rewritten as a stable v1 release, and supports defining PostgreSQL connection parameters in the official PostgreSQL Airflow hook. See [the documentation](https://great-expectations.github.io/airflow-provider-great-expectations/latest/getting-started/#manage-data-source-credentials-with-airflow-connections) for more details.
